src/ServiceInsight/SequenceDiagram/ModelCreator.cs

```diff
@@ -98,6 +99,19 @@ void Initialize()
 
                 sendingHandler.Out = sendingHandler.Out.Concat(new[] { arrow }).OrderBy(a => a).ToList();
             }
+
+            ReOrderHandlers(firstOrderHandlers);
+        }
+
+        private void ReOrderHandlers(IList<Handler> orderedHandlers)
```

Comment:
This method has mixed responsibilities:

- order the handlers
- assign the `handlers` field

Can we remove this method and inline the code in `Initialize`? It's only 4 statements, so I'm having a hard time justifying its existence.